src/main/java/com/google/cloud/tools/Option.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools;
+
+/**
+ * Contains a flag short ("-f") and long ("--force") forms, along with their type.
+ */
+public enum Option {
```

Comment:
Why is that important here?

Conceptually, any option can be meaningful across many actions. You should refer to the actions themselves to know which options one action takes. (The acceptedFlags field)